internal/server/v1/propertyvalues/golden/bulk_property_values_in/containedIn1.json

```diff
@@ -58001,5 +62129,5 @@
       ]
     }
   ],
-  "nextToken": "H4sIAAAAAAAA/4TQwUrDQBCAYRNDSbYW4p5y8hVaD4qgHtKtlngKxF68yLoZ4sJmBzYTJG8vaS8i2+Q4h+9nZljFlgp7S25YH6qcpQotSW2hLmxppAJ2I8B2fSfQwVZ2UFckSXeklTS5A8nDNOBRHKQBD+OQR/FlGrDtXHQlsG170rb5QAvexutc43oPuHe6Pk6ftztovJ18rnNVlEKcIncbb+KZxQ1gUa839x6/PD3o3UlFXv40ySOhaeBRGmQJX4wyS452MdosYW+TOnsx0IIl6YZKfSOane7I6TObPEy2WIk/4EojLfELHsZ/5eOkXJX9l9HqQNqMx/zDvwAAAP//AQAA///Pf2GPZAIAAA=="
+  "nextToken": "H4sIAAAAAAAA/4TQwUrDQBCAYZOGst1aiHvKyVdoPSiCemi3WuopEHvxIutmiAObXdhMlLy9JEEQiclxDt8/w/CML7WrLflmfcq2PNbOkkIL+dGmRmnglxJsVVfSedipCvKMFGFFqJXZelAijAMRsSAORMhCEbFZHPDdVHQlXVnWhLZ4dRYGG09TjYsDuIPHvJvervZQDHa2U53zYyplH7neDCYeOCvAHfP15mbAL/sHvXilaZDfj/JIIjUiioNkIeatTBadnbc2WfDnUZ08GijBkvJNpj+cM3usyOM/l9yOtnjqvsCnRlkSZyJkv+XdqFylHj8VQX9Ct/qHh2w2iet3g/pEaNpP/Nn8DQAA//8BAAD//87C0MmhAgAA"
```

Comment:
For my own understanding: Does `nextToken` change each time the API is called? If so, would that break tests?